src/theme/styles.module.scss

```diff
@@ -41,3 +41,11 @@
     }
   }
 }
+
+.img {
+  max-width: 700px !important;
```

Comment:
gross !important but ok :P

Reply:
Very 🫠 but it was either this or ejecting a whole new component from the layouts